core/src/main/java/com/opensymphony/xwork2/ognl/SecurityMemberAccess.java

```diff
@@ -81,8 +81,9 @@ public void restore(Map context, Object target, Member member, String propertyNa
     @Override
     public boolean isAccessible(Map context, Object target, Member member, String propertyName) {
         LOG.debug("Checking access for [target: {}, member: {}, property: {}]", target, member, propertyName);
-        
-        Class targetClass = target.getClass();
+
+        // target can be null in case of accessing static fields, since OGNL 3.2.8
+        Class targetClass = target != null ? target.getClass() : member.getDeclaringClass();
         Class memberClass = member.getDeclaringClass();
```

Comment:
As a small improvement you could switch L:86 and L:87, declare `Class memberClass` as `final` and reuse `memberClass` within the ternary expression.

Reply:
Thank you, done :)